### DISCUSSION TITLE
Installation issues: nothing happens when I type webots

### DISCUSSION THREAD

**Discussion by yvindo** [cyberbotics/webots#5456] [Q&A]
I have followed the Windows installation guide and gone through every step. In the build step, I have run the make command. The guide the specifies to type "webots" to launch it, but when I do this nothing happens. Have I done anything wrong in the installation, or is there anything else I have to do to launch?

**Comment by omichel** [comment]
Did the compilation of Webots went successful? Do you have a binary named `webots-bin.exe` in the `webots/msys64/mingw64/bin` folder?

**Reply by yvindo** [reply]
Yes, I see the executable is there. However, when I run it, I get several errors saying that the code can't continue running because lbgcc_s_seh-1.dll, libopenal-1.dll, libstdc++-6.dll and libfreetype-6.dll were not found. The errors suggested running the installation again could help, but it made no difference,

**Comment by omichel** [comment]
It seems your PATH environment variable is missing the path to these libraries which should be installed on your system, normally in the `C:\msys64\mingw64\bin` folder. Can you check this?

**Reply by yvindo** [reply]
None of the not found files are installed in the `C:\msys64\mingw64\bin` folder or anywhere else as far as I can see..

**Reply by omichel** [reply]
So, where did you install MSYS2 as instructed in the [wiki](https://github.com/cyberbotics/webots/wiki/Windows-installation/#msys2-development-environment-and-git)?

**Reply by yvindo** [reply]
The MSYS2 program is in the `C:/msys64` folder. But I am a bit confused because there is another msys64 folder inside `/C/msys64/home/oyvin/webots/`, and this is where `webots-bin.exe` is located. So the location of the executable is in the `/C/msys64/home/oyvin/webots/msys64/mingw64/bin` folder.  I can see now that the missing files are all in the `/C/msys64/mingw64/bin` folder. Does this make sence at all, or should `webots-bin.exe` also be in the outer folder?

**Reply by omichel** [reply]
Yes, this is all correct. You should simply ensure that your PATH environment variable includes at least these two folders.
This is normally done automatically in this [script](https://github.com/cyberbotics/webots/blob/master/scripts/install/msys.sh) called by [~/.bash_profile](https://github.com/cyberbotics/webots/blob/master/scripts/install/bash_profile.windows).

**Reply by yvindo** [reply]
It seems the paths were not automatically added by the script, so I added them manually, but this didn't make any change in the errors. I added `C:\msys64\mingw64\bin` and `C:\msys64\home\oyvin\webots\msys64\mingw64\bin` to the PATH environment variable. Do I need to run anything again?

**Reply by omichel** [reply]
Here is what I have on my development machine:
![_C_msys64_home_Olivier_webots 11_11_2022 13_58_28](https://user-images.githubusercontent.com/1264964/201345482-c840d599-7a76-4a11-a999-fdfadb6c93b1.png)
